### ISSUE TITLE
Employee registration

### ISSUE THREAD

**Issue by SamuelRocha2015** [SamuelRocha2015/travel-manager-api#13]
**User Story**

As an user i want to register as a employee, so that way create travels requests for myself.

**Acceptance criteria**

- To register an employee using any unauthenticated profile.

**Comment by QueirozDavi** [comment]
"To register an employee using an employee profile."
At first a employee don't have a profile setted, so that what a employee can't register himself with a employee profile,  we have to check if the person who is trying make a register have a profile and check that profile is administrator, if not a administrator user is trying make a employee register, the system would launch a exception (only for facilitator or manager).
So in short, only an administrator or a user without a profile can register a employee

**Comment by QueirozDavi** [comment]
for that issue i prefer to use a endpoint who a user with out a register can create a employee profile
